pkg/module_manager/go_hook/patchable_values.go

```diff
@@ -1,25 +1,63 @@
 package go_hook
 
 import (
+	"encoding/json"
+	"fmt"
 	"strings"
 
-	"github.com/Jeffail/gabs"
+	"github.com/tidwall/gjson"
 
 	"github.com/flant/addon-operator/pkg/utils"
 )
 
 type PatchableValues struct {
-	Values          *gabs.Container
+	values          *gjson.Result
 	patchOperations []*utils.ValuesPatchOperation
 }
 
 func NewPatchableValues(values map[string]interface{}) (*PatchableValues, error) {
-	gabsContainer, err := gabs.Consume(values)
+	data, err := json.Marshal(values)
 	if err != nil {
 		return nil, err
 	}
+	res := gjson.ParseBytes(data)
 
-	return &PatchableValues{Values: gabsContainer}, nil
+	return &PatchableValues{values: &res}, nil
+}
+
+// Get value from patchable. It could be null value
+func (p *PatchableValues) Get(path string) gjson.Result {
+	return p.values.Get(path)
+}
+
+// GetOk returns value and `exists` flag
+func (p *PatchableValues) GetOk(path string) (gjson.Result, bool) {
+	v := p.values.Get(path)
+	if v.Exists() {
+		return v, true
+	}
+
+	return v, false
+}
+
+// GetRaw get empty interface
+func (p *PatchableValues) GetRaw(path string) interface{} {
```

Comment:
Do we have a use case for this method? I think it will be rarely used.

Reply:
It is really rarely used. Just to avoid calls like x.Values.Values().(map[string]interface{}). But its cheaper to have it then to add again in future when we need it